app/src/main/java/com/sdp/movemeet/backend/BackendManager.java

```diff
@@ -23,7 +23,16 @@
      * @param path the path of the instance in the backend.
      * @return a Task, the success of which determines the success of the operation.
      */
-    Task<?> set(T object, String path, String field, String value);
+    Task<?> set(T object, String path);
+
+    /**
+     * Update (i.e. add) a String value to a Firebase Firestore array field
+     * @param value the String value to update in the array.
+     * @param path the path of the instance in the backend.
+     * @param field the field to which the array corresponds in the backend.
+     * @return a Task, the success of which determines the success of the operation.
+     */
+    Task<?> updt(String value, String path, String field);
```

Comment:
Why not call it `update`?

Reply:
Also, I would switch the parameter order to path - field - value, but that's just my personal preference

Reply:
That's a good point, I will modify it straight away!